db/migrate/20180131010756_create_imageurls.rb

```diff
@@ -1,7 +1,7 @@
 class CreateImageurls < ActiveRecord::Migration[5.1]
   def change
     create_table :imageurls do |t|
-      t.string :imageurl
+      t.string :imageurl, null: false
```

Comment:
We should avoid modifying existing migration that are not created inside this PR, because the production system only migrate upwards without rebuilding the db.  So, any change to migrations that are already ran will not be picked up.

Reply:
What we can do is to add a new migration to add the not-null constraint.